src/AddTodoForm.js

```diff
@@ -24,14 +25,9 @@ function AddTodoForm( { onAddTodo }) {
     }
     return (
         <form id="todoForm" onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle">Title</label>
-            <input 
-            name = "title" 
-            type = "text" 
-            id="todoTitle" 
-            value={todoTitle} 
-            onChange={handleTitleChange}>
-            </input>
+            <InputWithLabel isFocused todoTitle={todoTitle} handleTitleChange={handleTitleChange}>
+                Title
+            </InputWithLabel>
```

Comment:
Don't forget to set isFocused to true here and set the `id` attribute.

It would look something like this:
```suggestion
            <InputWithLabel isFocused={true} id={"whateveryouwant"} name={"not-needed-for-this-exercice-but-cool-to-have"} todoTitle={todoTitle} handleTitleChange={handleTitleChange}>
                Title
            </InputWithLabel>
```